src/raw/mod.rs

```diff
@@ -42,18 +40,34 @@ mod requests;
 /// **But, you should not need to worry about all this:** Many functions which accept a
 /// `ColumnFamily` accept an `Into<ColumnFamily>`, which means all of the above types can be passed
 /// directly to those functions.
-#[derive(Default, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
-pub struct ColumnFamily(String);
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+pub enum ColumnFamily {
+    Default,
+    Lock,
+    Write,
+    Version,
+}
 
 impl<T: Into<String>> From<T> for ColumnFamily {
     fn from(i: T) -> ColumnFamily {
-        ColumnFamily(i.into())
+        match i.into().as_str() {
+            "default" => ColumnFamily::Default,
+            "lock" => ColumnFamily::Lock,
+            "write" => ColumnFamily::Write,
+            "versioncf" => ColumnFamily::Version,
```

Comment:
the string representation for the Version CF is `"ver_default"` (sorry, that was my mistake). The variant should probably be named `VersionDefault`.